content/es/docs/reference/glossary/namespace.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Namespace
+id: namespace
+date: 2018-04-12
+full_link: /docs/concepts/overview/working-with-objects/namespaces
```

Comment:
```suggestion
full_link: /es/docs/concepts/overview/working-with-objects/namespaces/
```

Reply:
This is not already handled by Hugo?

I meant, if you point to the `en` version, and if the localised version is available, it will serve the localised one?

Reply:
That's an ambition but right now there aren't any rendering hooks configured to make that happen.

Reply:
Thanks for the clarification, the link has been updated in https://github.com/kubernetes/website/pull/22259/commits/891b912dff2a74085792d83b04e0fe0ab054fe6f.